src/devices/slacky_diy.ts

```diff
@@ -926,6 +933,98 @@ function waterPreset(): ModernExtend {
     return {toZigbee, exposes, isModernExtend: true};
 }
 
+const air_extend = {
+    led_brightness: (): ModernExtend => {
+        const exposes: Expose[] = [e.numeric("brightness", ea.ALL).withValueMin(0).withValueMax(255).withDescription("LED brightness")];
+
+        const toZigbee: Tz.Converter[] = [
+            {
+                key: ["brightness"],
+                convertSet: async (entity, key, value, meta) => {
+                    await entity.command(
+                        "genLevelCtrl",
+                        "moveToLevel",
+                        {level: value as number, transtime: 0},
+                        utils.getOptions(meta.mapped, entity),
+                    );
+                    return {state: {brightness: value}};
+                },
+                convertGet: async (entity, key, meta) => {
+                    await entity.read("genLevelCtrl", ["currentLevel"]);
+                },
+            },
+        ];
+
+        const fromZigbee: Fz.Converter[] = [
+            {
+                cluster: "genLevelCtrl",
+                type: ["attributeReport", "readResponse"],
+                convert: (model, msg, publish, options, meta) => {
+                    const result: KeyValueAny = {};
+                    if (Object.hasOwn(msg.data, "currentLevel")) {
+                        const data = Number.parseInt(msg.data.currentLevel, 10);
+                        result.brightness = data;
+                    }
+                    return result;
+                },
+            },
+        ];
+
+        return {
+            exposes,
+            fromZigbee,
+            toZigbee,
+            isModernExtend: true,
+        };
+    },
+
+    features_sensors: (): ModernExtend => {
+        const exposes: Expose[] = [
+            e.composite("features_sensors", "features_sensors", ea.SET).withFeature(e.enum("features", ea.STATE_SET, switchFeatures)),
+        ];
+
+        const toZigbee: Tz.Converter[] = [
+            {
+                key: ["features_sensors"],
+                convertSet: async (entity, key, rawValue, meta) => {
+                    const endpoint = meta.device.getEndpoint(1);
+                    // biome-ignore lint/suspicious/noExplicitAny: ignored using `--suppress`
+                    const value = (rawValue as any).features;
+                    if (value != null) {
+                        const lookup = {
+                            Nothing: 0,
+                            "CO2 forced calibration": 1,
```

Comment:
```suggestion
                            "co2_forced_calibration": 1,
```

Please make all snake case